airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2062,7 +2062,7 @@ def _job_id(self, context):
 
         exec_date = context['execution_date'].isoformat()
         job_id = f"airflow_{self.dag_id}_{self.task_id}_{exec_date}_{uniqueness_suffix}"
-        return re.sub(r"\:|-|\+\.", "_", job_id)
+        return re.sub(r"[:\-+.]", "_", job_id)
```

Comment:
Can you add test for this? It will help us avoid regression. Probably using parametrize will be good idea

Reply:
@turbaszek Test added.